.changeset/five-houses-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"vscode-apollo": patch
+---
+
+Only show "Run in Studio" gutter action for local graphs if an endpoint is configured.
```

Comment:
I'm not 100% sure.. it seems like a good idea, but might someone get use from a link to explorer that doesn't fill in the endpoint?